2025-09-18 14:50:00,871 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 14:54:00,880 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 14:58:00,929 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:02:01,009 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:06:00,093 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:10:00,146 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:14:00,169 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:18:00,245 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:24:43,339 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:26:00,366 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:44:04,871 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:46:00,958 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:50:01,018 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:54:00,063 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 15:58:00,082 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 16:02:00,114 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 16:06:00,191 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 16:10:00,285 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 16:14:00,329 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 16:18:00,342 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
2025-09-18 16:22:00,415 ERROR scheduler Exception in Enqueue Events for Site portfolio.local
Traceback (most recent call last):
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 117, in enqueue_events_for_site
    if is_scheduler_inactive():
       ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 161, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/scheduler.py", line 173, in is_scheduler_disabled
    scheduler_disabled = not frappe.get_system_settings("enable_scheduler")
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/core/doctype/system_settings/system_settings.py", line 231, in get_system_settings
    system_settings = frappe.client_cache.get_doc("System Settings")
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in get_doc
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 529, in get_value
    val = generator()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/utils/redis_wrapper.py", line 566, in <lambda>
    return self.get_value(key, generator=lambda: frappe.get_doc(doctype, name))
                                                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/utils/__init__.py", line 218, in wrapper
    return dispatch(args[0])(*args, **kw)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/document.py", line 123, in get_doc_str
    controller = get_controller(doctype)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 83, in get_controller
    site_controllers[doctype] = import_controller(doctype)
                                ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/model/base_document.py", line 94, in import_controller
    doctype_info = frappe.db.get_value("DocType", doctype, ("module", "custom", "is_tree"), as_dict=True)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 554, in get_value
    result = self.get_values(
    	doctype,
    ...<13 lines>...
    	wait=wait,
    )
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 667, in get_values
    out = query.run(as_dict=as_dict, debug=debug, update=update, run=run, pluck=pluck)
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/query_builder/utils.py", line 85, in execute_query
    result = frappe.local.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 248, in sql
    self.connect()
    ~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/database.py", line 146, in connect
    self.get_connection()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 109, in get_connection
    conn = self._get_connection()
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 114, in _get_connection
    return self.create_connection()
           ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/mohammednishamnk/Documents/works/startecht/apps/frappe/frappe/database/mariadb/mysqlclient.py", line 117, in create_connection
    return MySQLdb.connect(**self.get_connection_settings())
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/Users/mohammednishamnk/Documents/works/startecht/env/lib/python3.13/site-packages/MySQLdb/connections.py", line 200, in __init__
    super().__init__(*args, **kwargs2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
MySQLdb.OperationalError: (1045, "Access denied for user '_35acf0798e661fd5'@'localhost' (using password: YES)")
